src/lib/packager/clients/pkg_finish.rb

```diff
@@ -288,8 +289,15 @@ def sync_target_sources
     #
     # @see Yast::Packager::CFA::ZyppConf#set_minimalistic!
     def set_minimalistic_libzypp_conf
-      log.info("Setting libzypp configuration as minimalistic")
-      config = Packager::CFA::ZyppConf.new
+      config = if File.file?("/etc/zypp/zypp.conf")
+        log.info("Setting libzypp configuration as minimalistic")
+        Packager::CFA::ZyppConf.new
+      elsif File.file?("/etc/dnf/dnf.conf")
```

Comment:
I do not know if it makes sense to have dnf and zypper installed in parallel, but what about adapting both if they are present?